src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,190 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+
+    class Node<A> {
```

Comment:
1. should be private static
2. always try to declare access modifier explicitly
3. generic types are usually `T` - commonly accepted
4. inner classes should be located at the end of the class according to the Java convention
```suggestion
    private static class Node<T> {
```